index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <title>Fork Me! FCC: Test Suite Template</title>
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <nav id="navbar" class="nav">
+    <ul class="nav-list">
+      <li>
+        <a href="#welcome-section">About</a>
+      </li>
+      <li>
+        <a href="#projects">Work</a>
+      </li>
+      <li>
+        <a href="#contact">Contact</a>
+      </li>
+    </ul>
+  </nav>
+
+  <section id="welcome-section" class="welcome-section">
+    <h1>Hey we&apos;re</h1>
+    <p>web developers</p>
+  </section>
+
+
+  <section id="projects" class="projects-section">
+    <h2>Check out our projects</h2>
+
+    <div class="projects-list">
+      <a href="#" class="project project-tile">
+        <img class="project-image"
+          src="https://images.pexels.com/photos/6476808/pexels-photo-6476808.jpeg?cs=srgb&dl=pexels-mikael-blomkvist-6476808.jpg&fm=jpg"
+          alt="project" />
+        <p class="project-p">
+          Digital Marketing
+        </p>
+      </a>
+      <a href="#" class="project project-tile">
+        <img class="project-image"
```

Comment:
for semantics reasons, it's a best practice to wrap the main content inside the main tag, this would help screen readers and also improve SEO